src/Razor/src/Microsoft.CodeAnalysis.Razor.CohostingShared/HtmlDocumentServices/HtmlDocumentSynchronizer.cs

```diff
@@ -165,6 +174,6 @@ internal TestAccessor(HtmlDocumentSynchronizer instance)
         }
 
         public Task<SynchronizationResult> GetSynchronizationRequestTaskAsync(TextDocument document, RazorDocumentVersion requestedVersion)
-            => _instance.GetSynchronizationRequestTaskAsync(document, requestedVersion);
+            => _instance.GetSynchronizationRequestTaskAsync(document, requestedVersion, CancellationToken.None);
```

Comment:
Please consider making this `TestAccesspr` method take a `CancellationToken` and pass in the `DisposalToken` from unit tests. That can help avoid work started by a test continuing to run after the test is finished.